src/Applications/Applications/examples/v1.0/Add-MgApplicationKey.md

```diff
@@ -1,18 +1,30 @@
-### Example 1: {{ Add title here }}
-```powershell
-PS C:\> {{ Add code here }}
-
-{{ Add output here }}
+###Example 1
```

Comment:
@SteveWilkins1123, the title formatting is broken due to a missing whitespace:
```suggestion
### Example 1
```

Also, are you using a script to read the examples from the reference docs? If so, can we also include the title of the example. See example at [Get-MgDomainFederationConfiguration.md](https://github.com/msewaweru/msgraph-sdk-powershell/blob/0f1eeedd772a260ac93d75b7a98991dd30ae5d61/src/Identity.DirectoryManagement/Identity.DirectoryManagement/examples/v1.0/Get-MgDomainFederationConfiguration.md#example-1-get-the-federation-settings-for-a-federated-domain).

Reply:
That can't be done for 2 reasons. In a lot of cases the title is missing, and I am not reading them from the reference docs